### DISCUSSION TITLE
How to forward only specific messages to other chat?

### DISCUSSION THREAD

**Discussion by h45251880** [python-telegram-bot/python-telegram-bot#3750] [Q&A]
I want to fetch a specific message in a group and forward it to another chat.
I want to use regular expression to fetch.
For example, file_\s
I'm not sure if the API provides functionality.

**Comment by Bibo-Joshi** [comment]
Hey. In order for someone to be able to help you, you must ask a good technical question. Please read [this short article](https://github.com/python-telegram-bot/python-telegram-bot/wiki/Ask-Right) and try again ;)
